whois-commons/src/main/java/net/ripe/db/whois/common/DataSourceConfigurations.java

```diff
@@ -1,4 +1,4 @@
-package net.ripe.db.whois.common.source;
+package net.ripe.db.whois.common;
```

Comment:
why moving out of the `source` package?

Reply:
because rest of configs are in base ackage